.github/workflows/build.yaml

```
@@ -0,0 +1,68 @@
+name: CI/CD Pipeline
+
+on:
+  push:
```

Can we limit the docker build & push to changes on master branch only?
```suggestion
  push:
    branches:
      - main  
```